controllers/main.js

```diff
@@ -54,6 +55,10 @@ module.exports.updatePlayData = async (req, res, next) => {
     { sessionId: req.body.sessionId },
     { $push: { sessionData: guessData } }
   );
+  const playSessionData = await fetchPlaySessionData(req.user.username);
```

Comment:
I would recommend against using this practice of using something that was typed in by an user to interact with your database as that could be exploited